client/vfm-react-app/src/pages/Register.js

```diff
@@ -37,17 +45,52 @@ export const Register = () => {
     });
   };
 
+  React.useEffect(() => {
+   setCredentials({
+    user_id: Math.round(Math.random() * 10000) + 10,
+    username: credentials.first_name,
+    password: credentials.password,
+    first_name: credentials.first_name,
+    last_name: credentials.last_name,
+    address:"",
+    is_vendor: credentials.is_vendor === "farmer",
+    photo:null,
+    about_me:"",
+    email: credentials.email,
+    created_on: new Date(),
+    image_url:"",
```

Comment:
Some of these attributes will be filled by the profile modal that I just added last night. We should either delay the posting of this data until these fields are filled in with that modal or possibly create a PATCH endpoint for two requests.